doc/classes/TextEdit.xml

```diff
@@ -490,8 +506,10 @@
 			Undoes the previous action.
 		</constant>
 		<constant name="MENU_REDO" value="6" enum="MenuItems">
+			Redoes the previous action.
 		</constant>
 		<constant name="MENU_MAX" value="7" enum="MenuItems">
+
```

Comment:
The description should not be empty ;)
You can use something like "Represents the size of the [enum MenuItems] enum."